<?xml version="1.0"?>
<launch>
	<node pkg="fake_gravity_ft" type="fake_gravity_ft_node" name="fake_gravity_ft_node">
		<param name="source_frame" value="/world" />
		<param name="target_frame" value="/ft_sensor" />
		<param name="mass" value="2" />
		<param name="distance_y" value="0.1" />
	</node>
</launch>